stellar/README.md

```diff
@@ -173,6 +180,27 @@ node stellar/its.js set-trusted-address [chain-name] [trusted-address]
 ```
 
 #### Remove Trusted Address
+
 ```bash
 node stellar/its.js remove-trusted-address [chain-name]
 ```
+
+### Example
+
+#### GMP - Send
+
+```bash
+node stellar/gmp.js send [destination-chain] [destination-address] [payload] [gas-token-address] [gas-fee-amount]
```

Comment:
# approve should be done before executing the gmp call
soroban contract invoke --network testnet --id CDLZFC3SYJYDZT7K67VZ75HPJVIEUVNIXF47ZG2FB2RMQQVU2HHGCYSC --source-account wallet -- approve --from $CALLER_ADDRESS --expiration_ledger 3110400 --spender $GAS_SERVICE --amount 100